modules/setting/setting.go

```diff
@@ -647,6 +649,8 @@ func loadFromConf(allowEmpty bool, extraConfig string) {
 		}
 	}
 	EnableLetsEncrypt = sec.Key("ENABLE_LETSENCRYPT").MustBool(false)
+	ACMECAURL = sec.Key("ACME_CAURL").MustString("")
+	ACMECARoot = sec.Key("ACME_CARoot").MustString("")
```

Comment:
```suggestion
	ACMECAURL = sec.Key("ACME_CA_URL").MustString("")
	ACMECARoot = sec.Key("ACME_CA_ROOT").MustString("")
```

The underline makes the variables easier to read